### ISSUE TITLE
Rectangle could use a setCenter() function

### ISSUE THREAD

**Issue by BigZaphod** [photonstorm/phaser#4267]
Rectangle has setSize() and setPosition() functions, but no setCenter(). I just ran across a case where it would have been nice to have this instead of using the separate centerX and centerY properties, so I figured I'd mention it! :)

**Comment by halilcakar** [comment]
Isn't this setOrigin ?

**Comment by BigZaphod** [comment]
No. Rectangle doesn't have a function named `setOrigin` as far as I can tell. I want this function:

```javascript
    setCenter: function (x, y)
    {
        this.centerX = x;
        this.centerY = y;

        return this;
    }
```
